modules/repository-s3/src/main/config/log4j2.properties

```diff
@@ -12,3 +12,7 @@ logger.com_amazonaws_auth_profile_internal_BasicProfileConfigFileLoader.level =
 
 logger.com_amazonaws_services_s3_internal_UseArnRegionResolver.name = com.amazonaws.services.s3.internal.UseArnRegionResolver
 logger.com_amazonaws_services_s3_internal_UseArnRegionResolver.level = error
+
+logger.org_elasticsearch_entitlement_runtime_policy_PolicyManager.name = org.elasticsearch.entitlement.runtime.policy.PolicyManager.repository-s3
```

Comment:
Can we only change the specific aws sdk module that is problematic?

Reply:
It's hard to imagine wanting to get a warning about any module in this component, but I can add the module if you think that's better.